hollow/src/main/java/com/netflix/hollow/core/schema/SimpleHollowDataset.java

```diff
@@ -0,0 +1,66 @@
+/*
+ *
+ *  Copyright 2019 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+package com.netflix.hollow.core.schema;
+
+import com.netflix.hollow.api.error.SchemaNotFoundException;
+import com.netflix.hollow.core.HollowDataset;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * A HollowDataset implementation which only describes a set of schemas comprising a dataset. 
+ */
+public class SimpleHollowDataset implements HollowDataset {
```

Comment:
I wonder if there's a more descriptive name. Something is simple until it's simpler variant comes along.